src/app.js

```diff
@@ -25,6 +25,8 @@ const internals = {};
 const DEFAULT_GREETINGS_REGEX = /^(get started|good(morning|afternoon)|hello|hey|hi|hola|what's up)/i;
 const DEFAULT_HELP_REGEX = /^help\b/i;
 
+/*:: type Session = {count: number, profile: ?Object} */
```

Comment:
This documentation is incomplete. I only added what was necessary to get the linter to be quiet. So this is just an experiment in maintainability vs laziness